<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>146</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Page Shuffler</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QFilenameEdit" name="rectos_filename_edit"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="output_filename_label">
      <property name="text">
       <string>Sortie</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QFilenameEdit" name="output_filename_edit"/>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="output_filename_button">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>20</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="3">
     <widget class="QComboBox" name="mode_combobox">
      <item>
       <property name="text">
        <string>Rectos dans l'ordre, versos inversés</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Rectos et versos dans l'ordre</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="rectos_filename_label">
      <property name="text">
       <string>Rectos</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="versos_filename_label">
      <property name="text">
       <string>Versos</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="3">
     <widget class="QPushButton" name="start_button">
      <property name="text">
       <string>Démarrer</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="versos_filename_button">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>20</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="rectos_filename_button">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>20</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QFilenameEdit" name="versos_filename_edit"/>
    </item>
    <item row="2" column="0" colspan="3">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="3">
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QFilenameEdit</class>
   <extends>QLineEdit</extends>
   <header>qfilenameedit.h</header>
   <slots>
    <signal>dropped(QDropEvent*)</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>rectos_filename_edit</tabstop>
  <tabstop>rectos_filename_button</tabstop>
  <tabstop>versos_filename_edit</tabstop>
  <tabstop>versos_filename_button</tabstop>
  <tabstop>output_filename_edit</tabstop>
  <tabstop>output_filename_button</tabstop>
  <tabstop>mode_combobox</tabstop>
  <tabstop>start_button</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>rectos_filename_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>rectos_filename_button_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>293</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>177</x>
     <y>27</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>versos_filename_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>versos_filename_button_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>293</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>177</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>output_filename_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>output_filename_button_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>293</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>177</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rectos_filename_edit</sender>
   <signal>dropped(QDropEvent*)</signal>
   <receiver>MainWindow</receiver>
   <slot>rectos_filename_dropped(QDropEvent*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>62</x>
     <y>13</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>30</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>versos_filename_edit</sender>
   <signal>dropped(QDropEvent*)</signal>
   <receiver>MainWindow</receiver>
   <slot>versos_filename_dropped(QDropEvent*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>223</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>10</x>
     <y>5</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>start_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>start_button_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>168</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>versos_filename_button_clicked()</slot>
  <slot>rectos_filename_button_clicked()</slot>
  <slot>output_filename_button_clicked()</slot>
  <slot>rectos_filename_dropped(QDropEvent*)</slot>
  <slot>versos_filename_dropped(QDropEvent*)</slot>
  <slot>start_button_clicked()</slot>
 </slots>
</ui>
